### PR TITLE
Code fix for K8s fetch failure

### PR DESCRIPTION
**Pull request by hprateek43**
Previous code was not working. A final fix to make this repo working and then we can freeze this repo. Request owner to merge this change as it fixes previous bugs.

### CODE REVIEW

**Review by hprateek43 — COMMENTED** [review]
k8s_extensions_v1beta1_deployment: Failed to patch object: unrecognized type: string #22.

Fixes this bug

File: README.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-# DEPRECATED
```
Comment (fabianvf) [inline README.md:1]: Could you add this section back?

File: ansible-kubernetes-modules.spec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Name:           ansible-kubernetes-modules
```
Comment (fabianvf) [inline ansible-kubernetes-modules.spec:1]: This file as well, I don't think we should be reverting the changelog

File: module_utils/k8s_common.py [hunk 3]
```diff
@@ -162,12 +162,12 @@ def execute_module(self):
             if self.helper.base_model_name_snake.endswith('list'):
                 # For list modules, execute a GET, and exit
                 k8s_obj = self._read(name, namespace)
-                return_attributes[self.kind] = k8s_obj.to_dict() if k8s_obj else {}
```
Comment (fabianvf) [inline module_utils/k8s_common.py:165]: Were these removed on purpose? I think `k8s_obj` has a chance of being `None` which is why these guards were put in place.

**Review by fabianvf — CHANGES_REQUESTED** [review]
I don't have any real objection to this change, I think there were some changes brought in that are unrelated though. 

Just FYI this repo is deprecated in favor of the `k8s` module that shipped in Ansible 2.6 (which is the combined `openshift_raw` + `k8s_raw`  from 2.5)

**Comment by gundalow** [comment]
Deprecated, please use the `k8s` modules that ship with Ansible

**Pull request closed without merging**